### PR TITLE
Adjust to recent go-makefile-maker changes

### PR DESCRIPTION
**Pull request by SuperSandro2000**

### CODE REVIEW

**Comment by notque** [comment]
@SuperSandro2000 I don't understand how to make sure the make generate is ran in the ci workflow via the go-makefile-maker config. Can you assist here?

**Comment by notque** [comment]
I guess one option is to turn off go makefile maker's control of the ci workflow, and just manually have the ci workflow.

that seems reasonable.

File: Makefile.maker.yaml [hunk 1]
```diff
@@ -27,11 +38,12 @@ verbatim: |
   static-check: generate
```
Comment (SuperSandro2000) [inline Makefile.maker.yaml:38]: I think we need this also for build-all

```suggestion
  build-all: generate
  static-check: generate
```

**Comment by SuperSandro2000** [comment]
> I guess one option is to turn off go makefile maker's control of the ci workflow, and just manually have the ci workflow.
> 
> that seems reasonable.

Yay and nay. Then you need to bump the go version in multiple places. The go-setup workflow is usually behind with the default latest version it uses from it's own cache but that can be overwritten by setting the go version explicitly. And generally having to keep those things in sync is kinda a pain.

**Comment by notque** [comment]
Doing a branch deploy to validate everything is functional, if so I will merge.

**Comment by notque** [comment]
Tested in QA, works. Merging.

**Pull request merged**